### ISSUE TITLE
Inconsistent `us-west-1` availability zones

### ISSUE THREAD

**Issue by SeanBickle** [spulec/moto#5494]
`us-west-1` only has [2 availability zones accessible to customers](https://aws.amazon.com/about-aws/global-infrastructure/regions_az/). The [mapping of availability zone name to availability zone ID](https://docs.aws.amazon.com/ram/latest/userguide/working-with-az-ids.html) is per-account:

> AWS maps the physical Availability Zones randomly to the Availability Zone names for each AWS account

For `us-west-1`, the two valid availability zone IDs are `usw1-az1` and `usw1-az3`. As far as I can tell, it doesn't really matter which availability zone name is mapped to which availability zone ID.

[`moto/ec2/models/availability_zones_and_regions.py`](https://github.com/spulec/moto/blob/master/moto/ec2/models/availability_zones_and_regions.py) defines the mock availability zones for each region. Note that `us-west-1a` and `us-west-1b` are defined, and these are correctly mapped to the above availability zone IDs.

However, [`moto/ec2/resources/instance_type_offerings/availability-zone/us-west-1.json`](https://github.com/spulec/moto/blob/master/moto/ec2/resources/instance_type_offerings/availability-zone/us-west-1.json) references availability zones `us-west-1b` and `us-west-1c`.

These need to be consistent. Otherwise, it is possible to create a subnet in `us-west-1a` but subsequently fail to find a corresponding instance type offering for the subnet. I.e.:

```python
>>> subnet_az = client.describe_subnets()['Subnets'][0]['AvailabilityZone']
>>> subnet_az
'us-west-1a'
>>> client.describe_instance_type_offerings(LocationType='availability-zone', Filters=[{'Name': 'location', 'Values': [subnet_az]}]
{'InstanceTypeOfferings': [], 'ResponseMetadata': {'RequestId': 'f8b86168-d034-4e65-b48d-3b84c78e64af', 'HTTPStatusCode': 200, 'HTTPHeaders': {'server': 'amazon.com'}, 'RetryAttempts': 0}}
```

**Comment by bblommers** [comment]
Thanks for raising this @SeanBickle. The instance-type-offerings are a direct dump from AWS - strange that they are using `1b` and `1c`, but not `1a`. Marking it as a bug to fix our hardcoded zones to also use `1b` and `1c` though.

**Comment by bblommers** [comment]
FYI @SeanBickle  I've decided to keep availability zones `us-west-1a` and `us-west-1b`, and simply renamed all the instance type offerings to use those zones. IMO that is the change with the lowest impact, and users can continue to use `describe_availability_zones` and expect the same output.

This is available in Moto 4.0.6, released just now.
